.github/workflows/discourse-plugin.yml

```diff
@@ -109,13 +110,17 @@ jobs:
           puts "Detected test types: #{matrix.inspect}"
 
           File.write(ENV["GITHUB_OUTPUT"], "matrix=#{matrix.to_json}")
+          if matrix.any?
+            File.write(ENV["GITHUB_OUTPUT"], "has_tests=true")
```

Comment:
`File.write` doesn't append, does it? (i.e. it will replace the `matrix` entry?)

Reply:
🤦 fixed